src/components/search/SearchOptions.js

```diff
@@ -26,25 +26,25 @@ const SearchOptions = () => {
                     </div>
                 </Col>
                 <Col sm={2}>
-                    <Form.Control as="select" id="selectSizeResultsCount" name="selectSizeResultsCount"
```

Comment:
@ianlongden This looks like an actual change, not just cleaning up comments, so I don't know if that's on purpose from a different story, or an issue with rebasing.  When doing some recent API stuff, it was getting rid of some stuff that Shuai was working, so she rebased it.  So checking if this is your change on purpose or if it should talk to someone else about it.

Reply:
Also the other changes on this file

Reply:
Sorry yes this is a change, we were getting warnings about 'selected' in pull downs and it said to declare this in the above with 'value'.  warnings now gone and we have the same functionality.

Reply:
Ah, so is 'value' just the starting value, and changing something into one of the options updating it ?  If the functionality works for changing from the dropdown, submitting, and getting the correct value instead of the original value, then cool

Reply:
We get the correct number of refs shown but the pull down does not reflect this , so something is awry.

Reply:
Fixed, needed too use defaultValue not value.

Reply:
Also for sortByPublishedDate ?